db/languages.yml

```diff
@@ -136,5 +136,5 @@ j:
   :interpreter_command: '%{interpreter} %{source}'
   :variants:
     j8:
-      :name: "J 8.0.3"
+      :name: "J 8.0"
```

Comment:
```suggestion
      :name: "J 8.03"
```

Reply:
Hmmm. Going by offical docs seems a good benchmark. They use a weird numbering system. `J803` would be the proper way to do it as you say. However, if I google `J803` the first result is for Java whereas if I google `J 8.03` I get the J (programming language) wikipedia page, which is much more reasonable. So, although this wasn't the change I was hoping for with this PR, I agree with you